clang-tools-extra/clang-tidy/rename_check.py

```diff
@@ -311,7 +315,7 @@ def main():
                 "Check name '%s' not found in %s. Exiting."
                 % (check_name_camel, cmake_lists)
             )
-            return 1
+            sys.exit(1)
```

Comment:
Technically, this is a functional change. `return 1` was not doing anything beyond `return`. Exit with code 1 instead.